internal/certs/certs.go

```diff
@@ -0,0 +1,153 @@
+package certs
+
+import (
+	"crypto/x509/pkix"
+	"fmt"
+	"github.com/openshift/library-go/pkg/crypto"
+	"github.com/pkg/errors"
+	"k8s.io/apiserver/pkg/authentication/user"
+	"k8s.io/client-go/tools/clientcmd"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+	"path/filepath"
+	"time"
+)
+
+type CertManager struct {
+	CertificatesDir string
+	userCert        []byte
+	userKey         []byte
+	caBundle        []byte
```

Comment:
```suggestion
	clusterCABundle        []byte
```

(to be more consistent with kubeconfig naming - the "ca bundle" goes under cluster, so it should be made clear in our naming